cvat/apps/documentation/user_guide.md

```diff
@@ -24,97 +24,157 @@ There you can:
+__Use LFS__. If the annotation file is large, you can create a repository with [LFS](https://git-lfs.github.com/) support.
 
-    __Segment size__. Use this option to divide a huge dataset into a few smaller segments.
+__Source__. To create huge tasks please use ``shared`` server directory (choose ``Share`` option in the dialog).
 
-    __Image Quality__. Use this option to specify quality of uploaded images. The option makes it faster to load high-quality datasets. Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
+__Flip images__. All selected files will be turned around 180.
 
-    Push ``Submit`` button and it will be added into the list of annotation tasks. Finally you should see something similar to the figure below:
+__Z-Order__. Defines the order on drawn polygons. Check the box for enable layered displaying.
 
-    ![](static/documentation/images/image006.jpg)
+__Overlap Size__. Use this option to make overlapped segments. The option makes tracks continuous from one segment into another. Use it for interpolation mode. There are several options for using the parameter:
+- For an interpolation task (video sequence). If an object exists on overlapped segments, the overlap is greater than zero and the annotation is good enough on adjacent segments, it will be automatically merged into a single track. If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file, you will have several tracks, one for each segment, which correspond to the object.
+- For an annotation task (independent images). If an object exists on overlapped segments, the overlap is greater than zero and the annotation is good enough on adjacent segments, it will be automatically merged into a single track. If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file, you will have several bounding boxes for the same object.
+Thus, you annotate an object on the first segment. You annotate the same object on second segment, and if you do it right, you will have one track inside your annotation file. If annotations on different segments (on overlapped frames) are very different or overlap is zero, you will have two tracks for the same object. This functionality works only for bounding boxes. Polygon, polyline, points don't support automatic merge on overlapped segments even the overlap parameter isn't zero and match between corresponding shapes on adjacent segments is perfect.
```

Comment:
The whole description isn't presice. If you annotate a bounding box on two adjusted segments they will be merged into one bounding box.